POLICY.rst

```diff
@@ -0,0 +1,104 @@
+=================================
+Salt Project maintenance policies
+=================================
+
+This document explains the current project maintenance policies. The goal of
+these policies are to reduce the maintenance burden on core maintainers of the
+Salt Project and to encourage more active engagement from the Salt community.
+
+* `Issue management`_
+* `Pull request management`_
+* `Salt Enhancement Proposals (SEP) process`_
+
+
+Issue management
+================
+Issues for the Salt Project are critical to Salt community communication and to
+find and resolve issues in the Salt Project. As such, the issue tracker needs to
+be kept clean and current to the currently supported releases of Salt. They also
+need to be free of feature requests, arguments, and trolling.
+
+We have decided to update our issue policy to be similar to RedHat community
+project policies.
+
+Community members who repeatedly violate these policies are subject to bans.
+
+#. All issues that were not opened against a currently supported release of Salt
+   will be closed.
+
+   - When an old release of Salt is marked out of support, all issues opened
+     against the now defunct release will be closed.
+   - If the issue is still present in the current release of Salt, submit a new
+     issue. Do not re-open the old issue after it has been closed.
+   - When opening a new issue that was a bug in a previous release of Salt, you
+     must validate it against a currently supported release of Salt for
+     consideration. Issues that do not show the problem against a current
+     release will be closed without consideration.
+#. Only defects can be submitted to the issue tracker.
+
+   - Feature requests without a PR will be immediately closed.
+   - Feature requests must be designated as a feature being developed and owned
+     by the issue submitter and assigned to a release. Otherwise they will be
+     immediately closed.
+   - Discussions about features can be held in the GitHub
+     `Discussions <https://github.com/saltstack/salt/discussions>`_ tab or in
+     the community `Open Hour <https://saltproject.io/calendar/>`_.
+   - Questions will be immediately closed.
+#. Issues must submit sufficient information.
+
+   - Issues must follow the relevant template for information.
+   - Issues that do not give sufficient information about the nature of the

```

Comment:
Wondering if this should state as follows:

Issues that do not give sufficient information about the nature of the issue **and how to reproduce the issue** will be immediately closed.

Change in **BOLD**

Reply:
I can add that. Sorry that this can't be merged any more. I'll add it manually.